src/sdl_ui.cpp

```diff
@@ -786,6 +786,7 @@ void SdlUi::ProcessKeyDownEvent(SDL_Event &evnt) {
 			return;
 		}
 
+		[[gnu::fallthrough]];
```

Comment:
I had experimented with some "fall-through comments" as mentioned in the link I provided in the commit description, but it did not work.
Also MSVC warns about not recognizing `gnu` attributes (of course^^).
If we do not figure out a better solution, I will move the check for <kbd>Enter</kbd> into the `default` case instead.